packages/manager/modules/emailpro/src/domain/dkim-autoconfig/emailpro-domain-dkim-autoconfig.controller.js

```diff
@@ -144,4 +186,23 @@ export default class EmailProDomainDkimAutoconfigCtrl extends DkimAutoConfigurat
         this.services.$scope.resetAction();
       });
   }
+
+  getBodyText(status) {
+    switch (status) {
+      case this.GLOBAL_DKIM_STATUS.OK:
+        return 'emailpro_tab_domain_diagnostic_dkim_deactivation';
+      case this.GLOBAL_DKIM_STATUS.DISABLED:
+        return 'emailpro_tab_domain_diagnostic_dkim_activation';
+      case this.GLOBAL_DKIM_STATUS.NOT_CONFIGURED:
+        return this.domainDiag.isOvhDomain
+          ? 'emailpro_tab_domain_diagnostic_dkim_activation_ovhcloud'
+          : 'emailpro_tab_domain_diagnostic_dkim_activation_no_ovhcloud';
+      case this.GLOBAL_DKIM_STATUS.NOK:
+        return 'emailpro_tab_domain_diagnostic_dkim_configuration_nok';
+      case this.GLOBAL_DKIM_STATUS.IN_PROGRESS:
+        return '';
```

Comment:
Hello! You can remove these two lines, it will give the same result.

Reply:
Done